src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs

```diff
@@ -190,6 +196,44 @@ s! {
         pub ss_flags: ::c_int,
         pub ss_size: ::size_t
     }
+
+    pub struct vscr_t {
+        if #[cfg(target_arch = "powerpc64")] {
+            __pad: [::c_uint; 3],
+            vscr_word: ::c_uint,
+        } else {
+            vscr_word: ::c_uint,
+            __pad: [::c_uint; 3],
+        }
+    }
```

Comment:
glibc source: https://github.com/bminor/glibc/blob/master/sysdeps/unix/sysv/linux/powerpc/sys/ucontext.h#L96-L108

```c
/* Container for Altivec/VMX Vector Status and Control Register.  Only 32-bits
   but can only be copied to/from a 128-bit vector register.  So we allocated
   a whole quadword speedup save/restore.  */
typedef struct _libc_vscr
{
#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
	unsigned int __pad[3];
	unsigned int __ctx(vscr_word);
#else
	unsigned int __ctx(vscr_word);
	unsigned int __pad[3];
#endif
} vscr_t;
```


*Discussion:* I check the target to see if it is `powerpc64` or `powerpc64le`. If there is any better way to determine Big Endian or Little Endian, please feel free to let me know. Many thanks in advance